frontend/src/interfaces/IStates.ts

```diff
@@ -28,31 +21,41 @@ export const initialDetailsData:IDetailsData = {
     picture: "",
     id: "",
     alt: "",
-    model: Views.NEW
+    model: Views.NEW,
 }
 
 interface IBaseState {
-    pending:boolean
+    pending:boolean,
+    success: boolean,
+}
+
+export interface IEmployeeState extends IBaseState{
+    employees: IEmployee[],
+    current: IEmployee | undefined,
```

Comment:
Anfängerfrage meinerseits: was macht der pipe operator an dieser Stelle? current: IEmployee | undefined

Reply:
dies ist ein interface. Dient zur typisierung von Objekten. Der Wert current kann in diesem Fall also entweder des Types IEmployee sein oder undefined